ProcessList.c

```diff
@@ -553,13 +554,39 @@ void ProcessList_rebuildPanel(ProcessList* this) {
    bool foundFollowed = false;
 
    for (int i = 0; i < processCount; i++) {
-      Process* p = (Process*) Vector_get(this->processes, i);
-
-      if ( (!p->show)
-         || (this->userId != (uid_t) -1 && (p->st_uid != this->userId))
-         || (incFilter && !(String_contains_i(Process_getCommand(p), incFilter)))
-         || (this->pidMatchList && !Hashtable_get(this->pidMatchList, p->tgid)) )
-         continue;
+       Process* p = (Process*) Vector_get(this->processes, i);
+
+       if ( (!p->show)
+               || (this->userId != (uid_t) -1 && (p->st_uid != this->userId))
+               || (incFilter && !(String_contains_i(Process_getCommand(p), incFilter))) ) {
+           continue;
+       } else {
```

Comment:
If there's no way the true branch continues, there's also no reason for making the false branch its own block. Reduce level of indentation where ever possible.

Reply:
I removed the else block and de-indented the contents.  good catch.